generative/maisi/maisi_inference_tutorial.ipynb

```diff
@@ -0,0 +1,288 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    !python -c "import generative" || pip install monai-weekly[monai-generative]
</pre><p>Instead of using <code>pip install monai-weekly[monai-generative]</code> here, please directly install monai-generative, thanks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1741/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Thanks, done